contributors/devel/development.md

```diff
@@ -168,69 +278,87 @@ development environment, please [set one up](http://golang.org/doc/code.html).
 | 1.17 - 1.18    | 1.13.9      |
 | 1.18+          | 1.14.4      |
 
-Ensure your GOPATH and PATH have been configured in accordance with the Go
-environment instructions.
-
-#### Upgrading Go
+##### A Note on Changing Go Versions
```

Comment:
The original instructions are unfortunately out of date :(

We should probably remove this section, but can do that as a follow-up. The updated instructions would go in the sig-release handbook - https://github.com/kubernetes/sig-release/pull/1109#issuecomment-661968929.